packaging/tests/tas-cmis/src/test/java/org/alfresco/cmis/search/AbstractE2EFunctionalTest.java

```diff
@@ -0,0 +1,58 @@
+package org.alfresco.cmis.search;
+
+import org.alfresco.cmis.CmisWrapper;
+import org.alfresco.dataprep.SiteService.Visibility;
+import org.alfresco.utility.data.DataContent;
+import org.alfresco.utility.data.DataSite;
+import org.alfresco.utility.data.DataUser;
+import org.alfresco.utility.data.RandomData;
+import org.alfresco.utility.model.SiteModel;
+import org.alfresco.utility.model.UserModel;
+import org.alfresco.utility.network.ServerHealth;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.BeforeClass;
+
+@ContextConfiguration ("classpath:alfresco-cmis-context.xml")
+public abstract class AbstractE2EFunctionalTest extends AbstractTestNGSpringContextTests
+{
+    /** The number of retries that a query will be tried before giving up. */
+    protected static final int SEARCH_MAX_ATTEMPTS = 1;
```

Comment:
Why only a single attempt?

Reply:
So the builds don't take too long. 

How many attempts would be ideal?

Reply:
For comparison, see also: https://github.com/Alfresco/acs-packaging/blob/master/tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/SearchQueryService.java#L80 (retries every 1 sec for up to 20 sec) ?